tuftool/CHANGELOG.md

```diff
@@ -78,6 +93,7 @@ Major update: much of the logic in `tuftool` has been factored out and added to
 ### Added
 - Everything!
 
+[0.6.0]: https://github.com/awslabs/tough/compare/tuftool-v0.5.0...tuftool-v0.6.0
```

Comment:
We'll also need to bump the version of tuftool in `tuftool/Cargo.toml`

Reply:
Good catch.